docs/getting_started.md

```diff
@@ -14,10 +14,10 @@
         python ./ts_scripts/install_dependencies.py
         ```
 
-     - For GPU with Cuda 10.2. Options are `cu92`, `cu101`, `cu102`, `cu111`, `cu113`, `cu116`, `cu117`, `cu118`
+     - For GPU with Cuda 12.1. Options are `cu92`, `cu101`, `cu102`, `cu111`, `cu113`, `cu116`, `cu117`, `cu118`, `cu121`
```

Comment:
this list is long no? :P

Reply:
Actually, we should truncate this to the latest version. Good point. Anyone looking for 0.6.0 can look at the readme page corresponding to 0.6.0 tag